suites/squid/rados/tier-2_rados_test-stretch-mode-upgrade.yaml

```diff
@@ -0,0 +1,264 @@
+# Suite contains tests related to election strategy and Stretched mode
+# Use cluster-conf file: conf/quincy/rados/stretch-mode-host-location-attrs.yaml
```

Comment:
Please change this to say - `conf/squid/rados/stretch-mode-host-location-attrs.yaml`

Reply:
Made the changes as per the comments.